### PR TITLE
Prefix all classes with rz- #272

### PR DESCRIPTION
**Pull request by funny-vince**
Prefix all classes with rz- to avoid conflicts with other CSS classes (e.g. from an other library) [#272](https://github.com/angular-slider/angularjs-slider/issues/272)

I was trying to fix a bug on a friend's project and I realized that another `vertical` class was overriding the left property of the angularjs-slider `vertical` class. I came here to report the issue, but it was already reported so I decided to fix it. I prefixed `tick`, `vertical` and `selected` with rz- (now `rz-tick`, `rz-vertical` and `rz-selected`) to avoid further conflicts with those class names.

### CODE REVIEW

File: dist/rzslider.js [hunk 1]
```diff
@@ -764,7 +764,7 @@
             tick.tooltipPlacement = this.options.vertical ? 'right' : 'top';
           }
           if (this.options.showTicksValues) {
-            tick.value = this.getDisplayValue(value, 'tick-value');
+            tick.value = this.getDisplayValue(value, 'rz-tick-value');
```
Comment (ValentinH) [inline dist/rzslider.js]: This line should not be modified in my opinion. It's not about css class but it is a simple key that is used for the translate option to determine which value we are translating.

**Comment by ValentinH** [comment]
Except the comment above, this is a really good job. :)

**Comment by funny-vince** [comment]
I updated my pull request. Thanks for your feedback :)

**Comment by ValentinH** [comment]
Great job! :+1:

**Pull request merged**